src/components/Modal/Modal.tsx

```diff
@@ -40,13 +35,14 @@ type ModalContentProps = {
    *
    * If undefined, the first focusable element (usually the close button) will be used.
    *
-   * @example
+   * ```
    * const inputFieldRef = useRef();
    *
    * <Modal initialFocus={inputFieldRef}>
    *   ...
    *   <InputField ref={inputFieldRef} />
    * </Modal>
+   * ```
```

Comment:
nice

Reply:
@anniehu4 Instead of `@example` we can use code blocks in prop doc blocks , which render in storybook. This should fix that glitch we noticed earlier this week